### PR TITLE
Show thumbnail in resource dashboard

### PR DESCRIPTION
**Pull request by athurg**
The resource dashboard now only show an icon for all resources. Change it to show the thumbnail while it was an image.

show

### CODE REVIEW

File: web/src/less/resource-cover.less [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .resource-cover {
   @apply w-20 h-auto ml-auto mr-auto opacity-40;
+  height: 5rem;
+  overflow-y: hidden;
```
Comment (Zeng1998) [inline web/src/less/resource-cover.less:4]: use tailwindcss.

**Review by Zeng1998 — CHANGES_REQUESTED** [review]

File: web/src/components/ResourceCover.tsx [hunk 2]
```diff
@@ -36,7 +37,7 @@ const ResourceCover = ({ resource }: ResourceCoverProps) => {
   const resourceType = getResourceType(resource);
   switch (resourceType) {
     case "image/*":
-      return <Icon.FileImage className="resource-cover" />;
+      return <img className="resource-cover h-20 w-20" src={getResourceUrl(resource) + "?thumbnail=1"}></img>;
```
Comment (Zeng1998) [inline web/src/components/ResourceCover.tsx:40]: use `` instead of `</img>`

**Review by Zeng1998 — CHANGES_REQUESTED** [review]

File: web/src/components/ResourceCover.tsx [hunk 3]
```diff
@@ -36,7 +37,7 @@ const ResourceCover = ({ resource }: ResourceCoverProps) => {
   const resourceType = getResourceType(resource);
```
Comment (Zeng1998) [inline web/src/components/ResourceCover.tsx:37]:
```suggestion
  const resourceType = getResourceType(resource);
  const resourceURL = getResourceUrl(resource);
```

**Review by Zeng1998 — CHANGES_REQUESTED** [review]

**Review by Zeng1998 — APPROVED** [review]
LGTM.

@CorrectRoadH  Please take a look and check the style.

**Review by CorrectRoadH — APPROVED** [review]
LGTM

**Pull request merged**